File: 02_ProgrammingFundamentals/_01_BasicSyntaxConditionalStatementsAndLoops/_03_MoreExercises/_04_ReverseString.java
package _02_ProgrammingFundamentals._01_BasicSyntaxConditionalStatementsAndLoops._03_MoreExercises;

import java.util.Scanner;

public class _04_ReverseString {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        // Read input value
        String console = scan.nextLine();

        // Add variable
        StringBuilder rebuild = new StringBuilder();

        // Reverse string
        for (int i = console.length() - 1; i >= 0; i--) {
            char symbol = console.charAt(i);

            rebuild.append(symbol);
        }

        // Print result
        System.out.println(rebuild);

        scan.close();
    }
}

File: 24_C#Fundamentals/_01_BasicSyntaxConditionalStatementsAndLoops/_03_MoreExercises/_04_ReverseString/_04_ReverseString.cs
using System;
using System.Text;

public class _04_ReverseString
{
    public static void Main(string[] args)
    {
        // Read input value
        string console = Console.ReadLine();

        // Add variable
        StringBuilder rebuild = new StringBuilder();

        // Reverse string
        for (int i = console.Length - 1; i >= 0; i--)
        {
            char symbol = console[i];
            rebuild.Append(symbol);
        }

        Console.WriteLine(rebuild.ToString());
    }
}
